force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        Integer sumValue = null;
+        if(a != null && b != null)
+            sumValue = a + b;
+        return sumValue; // Replace null with the variable you used to store the result
```

Comment:
logic is correct. But use {} with an If statement or it gets confusing for the next dev to read your code